client/src/main/java/org/apache/rocketmq/client/impl/factory/MQClientInstance.java

```diff
@@ -255,19 +255,17 @@ public void start() throws MQClientException {
     }
 
     private void startScheduledTask() {
-        if (null == this.clientConfig.getNamesrvAddr()) {
```

Comment:
I think what @qqeasonchen says makes sense, but we have discuss what behavior we want to have.

To me, if addresses have been manually specified,
it makes sense to fetch name servers only if a call to manually specified servers fail. No need to do periodic calls. What do you think?